commit 34ffabe311cc4267dbb955d38b87905621c144b2
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Fri Jan 24 23:06:56 2020 +0000

    drm/i915: Remove 'prefault_disable' modparam
    
    The 'prefault_disable' modparam was used by IGT to prevent a few
    prefaulting operations to make fault handling under struct_mutex more
    prominent. With the removal of struct_mutex, this is not as important
    any more and we have almost completely stopped using the parameter. The
    remaining use in execbuf is now immaterial and can be dropped without
    affecting coverage.
    
    We must re-address the idea of fault injection though.
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
    Cc: Antonio Argenziano <antonio.argenziano@intel.com>
    Cc: Joonas Lahtinen <joonas.lahtinen@linux.intel.com>
    Acked-by: Jani Nikula <jani.nikula@intel.com>
    Link: https://patchwork.freedesktop.org/patch/msgid/20200124230656.687503-1-chris@chris-wilson.co.uk

diff --git a/drivers/gpu/drm/i915/i915_params.h b/drivers/gpu/drm/i915/i915_params.h
index ef4069645cb8..cb16410c2ada 100644
--- a/drivers/gpu/drm/i915/i915_params.h
+++ b/drivers/gpu/drm/i915/i915_params.h
@@ -71,7 +71,6 @@ struct drm_printer;
 	param(unsigned long, fake_lmem_start, 0, 0400) \
 	/* leave bools at the end to not create holes */ \
 	param(bool, enable_hangcheck, true, 0600) \
-	param(bool, prefault_disable, false, 0600) \
 	param(bool, load_detect_test, false, 0600) \
 	param(bool, force_reset_modeset_test, false, 0600) \
 	param(bool, error_capture, true, 0600) \